config/initializers/class.rb

```diff
@@ -0,0 +1,13 @@
+class Class
```

Comment:
I don't think we should monkey-patch `Class` for this purpose. Could we instead create a LowerCamelizeable (or something?) module for inclusion? Also I'd like to discuss changing the implementation so it's more explicit within the class it's used in.

For example, declare the "normal style" attributes, with their documenting comments:

```
# a short descriptor indicating limits on displaying or editing this email
attr_accessor :ui_control

# the date this email address came into use
attr_accessor :from_date

# the date this email address stopped being used
attr_accessor :to_date
```

And then something like this to create the camelized aliases

```
camel_alias :uiControl, :fromDate, :toDate
```

Which would provide a bit of a search-ability advantage if you were trying to track down where the uiControl method came from, for instance.

Reply:
Also, including both `camel_alias` and `camel_accessor` would be handy in the case of the date related methods where the `from_date` and `to_date` methods would be better as methods that just reference the internal @fromDate camel accessor